packages/ui/src/components/AccountDisplay.tsx

```diff
@@ -99,10 +102,12 @@ const AccountDisplay = ({
               />
             )}
             {!!sub && <span>{sub}</span>}
-            <NameStyled>{localName || mainDisplay}</NameStyled>
+            <NameStyled className="multisigName">{localName || mainDisplay}</NameStyled>
```

Comment:
This class name is used and set in another component `PureProxyHeaderView`. There's the chance that we change/remove the other component without realizing we actually change this one. Can we centralize the design here?

Reply:
Account Display is a generic component used in the other components and looks the same. For PureProxyHeaderView we need style adjustments. The properties are deeply nested so it's hard to target Name and Address, for that reason I created a class to change the styles. It will have no impact to other components if you remove PureProxyHeaderView

Reply:
Right, but if you remove the classe names in `AccountDisplay.tsx`, which is something we've done before, because we said that we should avoid the use of classNames, then it'll break. We've had the issue in the past, and I'd like to avoid it as much as possible. However, I do definitely get your point, and making things super complex.

Maybe adding a comment here would help, like "className used here for specific cases in the AccounHeader for instance". What do you think?

Reply:
For me, it looks like a regular thing and it's easy to detect if your style is off as you have visual feedback. But if you find this helpful, I added the comments in Account Display

Reply:
Thanks. I don't like that kind of comment either, but again, it's because we've broken this in the past. I can't find the PR now, but there was this case, where, for cleaning, we removed some `classNames`, and migrated them to styled components, and things just broke all over without being spotted in review, because it's super hard to spot.